tableau_utilities/scripts/merge_config.py

```diff
@@ -2,18 +2,32 @@
 from tableau_utilities.general.cli_styling import Color, Symbol
 from tableau_utilities.scripts.gen_config import load_csv_with_definitions, generate_config
 
+# Print Styling
+color = Color()
+symbol = Symbol()
```

Comment:
```suggestion
COLOR = Color()
SYMBOL = Symbol()
```
Same as above comment